src/monocraft.py

```diff
@@ -69,8 +71,9 @@ def generateFont():
 		lig.addPosSub("ligatures-subtable", tuple(map(lambda codepoint: charactersByCodepoint[codepoint]["name"], ligature["sequence"])))
 
 	print(f"Generated {len(ligatures)} ligatures")
-	monocraft.generate("../dist/Monocraft.ttf")
-	monocraft.generate("../dist/Monocraft.otf")
+	if not os.path.exists(OUTPUT_DIR): os.makedirs(OUTPUT_DIR)
```

Comment:
Any chance you could move the OUTPUT_DIR constant above this line since it isn't needed globally in the module? Otherwise, looks good, thanks for catching this problem!